register.php

```diff
@@ -197,7 +197,7 @@
         }
 
         // Gender
-        if(!($gender == "Male" || $gender == "Female")) {
+        if(!($gender == "Male" || $gender == "Female" || $gender == "Non-binary" || $gender == "None")) {

```

Comment:
Once you make the $genders array on `User`, you can do

`if(!in_array($gender, User::$genders))`

Reply:
done